test/suites/standard/validate-certificate-rotation.robot

```diff
@@ -0,0 +1,75 @@
+*** Settings ***
+Documentation       Tests related to upgrading MicroShift
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/microshift-process.resource
+Library             Collections
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Variables ***
+${USHIFT_HOST}                      ${EMPTY}
+${USHIFT_USER}                      ${EMPTY}
+${KUBE_SCHEDULER_CLIENT_CERT}       /var/lib/microshift/certs/kube-control-plane-signer/kube-scheduler/client.crt
+${OSSL_CMD}                         openssl x509 -noout -dates -in
+
+
+*** Test Cases ***
+Certificate Rotation
+    [Documentation]    Performs Certificate Expiration Rotation test
+    Cert Expiration Date Should Match    ${KUBE_SCHEDULER_CLIENT_CERT}    365
```

Comment:
This would be clearer if the "365" was a date instead of a duration. Let the test compute the expected values, then check, change the clock, check again (with a new expected value).

Reply:
how does this sounds:
1. `Cert Expiration Date Should Match` rename  "Certifcate Will Valid for One Year" , and remove the passed duration paramter
1.A  calculate  the date a year from now 
1.B - check if the passed  cert file matches  the outcome of 1.A.

2. change the clock - 
2.A - expect microshift reboot and cert regenerate
2.B - repeat 1.